test/http-server.test.ts

```diff
@@ -29,6 +30,24 @@ describe('HTTP Server', () => {
     expect(server.listening).toBe(false);
   });
 
+  it("should support https if cert + key options are supplied", async () => {
+    const server = new HttpServer(dummyHandler, {
+      key: fs.readFileSync("test/keys/localhost-key.pem"),
+      cert: fs.readFileSync("test/keys/localhost-cert.pem"),
+    });
+
+    await expect(server.start()).resolves.not.toThrow();
+
+    expect(server.listening).toBe(true);
+
+    const host = `https://127.0.0.1:${server.address().port}`;
+    const res = await request(host).get('/').trustLocalhost(true).expect(200);
+
+    expect(res.body).toEqual({
+      value: 'Dummy response',
+    });
+  });
```

Comment:
I think `await server.stop();` is missing here.

Reply:
Fixed!